whitespace not ignored in Containerfile

We should ignore whitespace (empty lines, or lines with whitespace) in the Containerfile.

If we don't we get:

```
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/nubificus/bima/internal/image.NewInstructionLine(...)
	/home/runner/_work/bima/bima/internal/image/operation.go:20
main.getOperations({0xffffd3c1058f, 0x8}, {0xffffd3c10578, 0x16})
	/home/runner/_work/bima/bima/cmd/bima/build.go:161 +0x4ec
main.buildImage({0xffffd3c1058f?, 0x74391f?}, {0xffffd3c10578?, 0x20?})
	/home/runner/_work/bima/bima/cmd/bima/build.go:181 +0x2c
main.bimaBuild(0x4000237380)
	/home/runner/_work/bima/bima/cmd/bima/build.go:105 +0xb4c
github.com/urfave/cli/v2.(*Command).Run(0x400017da20, 0x4000237380, {0x4000100ae0, 0x6, 0x6})
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.25.0/command.go:273 +0x73c
github.com/urfave/cli/v2.(*Command).Run(0x400017dce0, 0x40002372c0, {0x4000030070, 0x7, 0x7})
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.25.0/command.go:266 +0x948
github.com/urfave/cli/v2.(*App).RunContext(0x40001b21e0, {0x842218?, 0x4000148708}, {0x4000030070, 0x7, 0x7})
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.25.0/app.go:332 +0x568
github.com/urfave/cli/v2.(*App).Run(...)
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.25.0/app.go:309
main.main()
	/home/runner/_work/bima/bima/cmd/bima/main.go:34 +0xf8
```

--- a/cmd/bima/build.go
+++ b/cmd/bima/build.go
@@ -160,6 +160,13 @@ func getOperations(contextDir string, containerFile string) ([]image.BimaOperati
 	log.Tracef("Read following lines from Containerfile: %v", lines)
 	operations := []image.BimaOperation{}
 	for _, line := range lines {
+		line = strings.TrimSpace(line)
+		if line == "" {
+			continue
+		}
+		for strings.Contains(line, "  ") {
+			line = strings.ReplaceAll(line, "  ", " ")
+		}
 		log.Tracef("Creating bima operation from line %q", line)
 		instruction := image.NewInstructionLine(line)
 		operation, err := instruction.ToBimaOperation()

